### DISCUSSION TITLE
Tap and midi clock

### DISCUSSION THREAD

**Discussion by micahvdm** [alf45tar/PedalinoMini#442] [General]
Hey, just started to sponsor, but I’ve been following this project since the first version! I must that this is a phenomenal project, well done!

I’ve noticed that midi clock sync is not working yet, is this something on the cards for the near future?

**Comment by alf45tar** [comment]
You can give a try to the latest public version just released. It should works pretty well on wired connection. Wireless connections have too much latency.

**Reply by micahvdm** [reply]
awesome, I'll give it a go! looks like your github actions are failing due to bpi_leaf_s3. I get a lot of errors in vs code about the board not being found...Your uploader page also fails on 3.1.5 due to it not building correctly.

**Reply by alf45tar** [reply]
I noted. I am fixing it.

**Reply by micahvdm** [reply]
It works so damn well! Thanks so much! I’m now gonna start the conversion of my arduino MIDI controller to PedalinoMini!!!
